civictechprojects/static/css/partials/_MainFooter.scss

```diff
@@ -1,4 +1,5 @@
 .MainFooter-footer {
+  display: none;
```

Comment:
Instead of hiding via CSS, we should comment out the area where we render this element in the first place.  It's the same end result, but faster since the browser can skip reading the html.